Source/src/utils/FileSystem.cpp

```diff
@@ -14,24 +15,23 @@ Hachiko::ModuleFileSystem::ModuleFileSystem()
     PHYSFS_setWriteDir(".");
 }
 
-Hachiko::ModuleFileSystem::~ModuleFileSystem()
+Hachiko::FileSystem::~FileSystem()
 {
     PHYSFS_deinit();
 }
 
-bool Hachiko::ModuleFileSystem::Init()
+bool Hachiko::FileSystem::Init()
 {
     HE_LOG("Init virtual file system");
 
     CreateContext();
 
     char* write_path = SDL_GetPrefPath("Akita", "Hachiko");
     SDL_free(write_path);
-
-    return true;
+    return false;
```

Comment:
This return value is correct?

Reply:
Nope. But as is not a Module anymore, Application does not validate its return value. I can modifying it to return true, or change the function to return void. What is better for you?

Reply:
Well seeing btw

Reply:
it's fine for me if this function never is asserted

Reply:
Ok!